assets/js/blocks/mini-cart/frontend.ts

```diff
@@ -152,4 +152,18 @@ window.addEventListener( 'load', () => {
 			);
 		}
 	} );
+
+	const head = document.head || document.getElementsByTagName( 'head' )[ 0 ];
+	const style = document.createElement( 'style' );
+	const body = document.body || document.getElementsByTagName( 'body' )[ 0 ];
```

Comment:
Just curious, do you think this check of `document.body` is needed? In several other places of this file we assume `document.body` exists.

Reply:
My bad! I will fix it. I think we can also use `document.head` without a fallback.